packages/config/src/forkConfig/types.ts

```diff
@@ -10,6 +10,12 @@ export type ForkInfo = {
   prevForkName: ForkName;
 };
 
+/** Set of config values that frequently change across hard-forks */
+export type ConfigValue = {
```

Comment:
anyone has a better name for this type? if not the jsdoc might suffice

Reply:
hmmm not really a fan of creating a subset of config. 

Would rather keep it simple for the purpose of devnet-5 branch, and come up with a systematic way to deal with all config values in one place. 

Something like https://github.com/Consensys/teku/blob/master/ethereum/spec/src/main/java/tech/pegasys/teku/spec/SpecVersion.java#L156 but not as cumbersome

Reply:
I am fine with either as long as it makes it easier to maintain and less error prone for future hard forks which both solution solve

if we wanna keep diff minimal we can revert last 2 commits

Reply:
> if we wanna keep diff minimal we can revert last 2 commits

Yea I would prefer that. Can deal with the config issue in another PR outside of devnet-5 branch.